.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,12 @@
+name: golangci-lint
+on: [push, pull_request]
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```

Comment:
We should use version `v1.32` since this is the [latest](https://github.com/golangci/golangci-lint/tags).

Reply:
fix it. https://github.com/elimity-com/scim/pull/98/commits/dbd546b68cefe49e8161fd63f7de369cd7220c7f